README.md

```diff
@@ -679,7 +689,7 @@ Triggers `nodeExpanded` event; pass silent to suppress events.
 Returns an array of matching node objects.
 
 ```javascript
-$('#tree').treeview('findNode', ['Parent', 'text']);
+$('#tree').treeview('findNodes', ['Parent', 'text']);
```

Comment:
Not really but the correct method is “findNodes” and not “findNode”.
If I try this example I get: “No such method : findNode”